app/schemas/models/ai_document.schema.js

```diff
@@ -4,24 +4,14 @@ const _ = require('lodash')
 const AIDocumentSchema = c.object({
   title: 'AI Document',
   description: 'A generative AI document',
-  required: ['type'], // TODO: required properties (name? content? owner?)
-  default: {}
 })
 
 _.extend(AIDocumentSchema.properties, {
   type: { type: 'string', description: 'The file type (html, py, jpg, etc.)' },
-  owner: c.objectId(),
-  scenario: c.objectId(),
-  project: c.objectId(), // Scenario link, project link, both, neither?
-  created: c.date({ title: 'Created', readOnly: true }),
-  name: { type: 'string' },
-  content: c.object({}, {
-    text: { type: 'string', description: 'Text contents of this document' },
-    url: { type: 'string', format: 'file', description: 'File link to binary contents of this document' }
-  })
+  source: { type: 'string', description: 'The contents of the document' }
```

Comment:
if this is `contents` why do we rename it to `source` instead of keeping it as `content` ?

Reply:
I think `source` is better, to differentiate from any rendered output.